terraform/role-ci.tf

```diff
@@ -0,0 +1,98 @@
+# Example of how to use policies from terraform-aws-serverless to create
```

Comment:
How far off are we placing something like this in `canary` submodule or even `canary-role-ci` submodule (for after `canary`) in terraform-aws-serverless?

Reply:
There's a few things I'd need to figure out before introducing this to `terraform-aws-serverless`:
- can we automatically compose `assume_role_policy`s from multiple modules? Is it enough to put any role principals (e.g. `lambda.amazonaws.com`) in the main module instead, even if their relevant submodule isn't used?
- if not automatic, does it make sense to export the principals from each module and have the user combine them?

Reply:
Ticketed in https://github.com/FormidableLabs/terraform-aws-serverless/issues/53